test_util/helpers.py

```diff
@@ -37,6 +37,81 @@ def authenticate(self, cluster):
         logging.info('Authentication successful')
 
 
+class ApiClient:
+
+    def __init__(self, cluster, user, api_base, default_headers=None, ca_cert_path=None):
+        self.cluster = cluster
+        self.user = user
+        self.api_base = api_base
+        if default_headers is None:
+            default_headers = dict()
+        self.headers = default_headers
+        self.ca_cert_path = ca_cert_path
+        self.port_mapping = {
```

Comment:
Point taken on vetoing static member variables. We can return to my settable port mapping when I actually implement an API that would use it.